client/components/payment-activity/payment-activity-data.tsx

```diff
@@ -76,8 +76,8 @@ const PaymentsActivityData: React.FC = () => {
 						type_is: 'refund',
 					} ) }
 				/>
-				<PaymentsDataTile
-					id="wcpay-payments-data-highlights__disputes"
+				<PaymentDataTile
+					id="wcpay-payment-data-highlights__disputes"
```

Comment:
I ❤️ the way you've corrected this in all the under-the-hood places too (CSS classes, variable names etc) 🙌🏻

Reply:
Thanks, @haszari 🙌 . There is one more change on the server side for which I will create a low priority issue and assign it to myself. It will require a very small but simultaneous change on the server and in the client. I will discuss this with the team regarding the best time and way to do it, and have it actioned.

Reply:
I have the PR on the server ready :) - https://github.com/Automattic/woocommerce-payments-server/pull/5146 

I have pushed the related changes for client on this PR too 
https://github.com/Automattic/woocommerce-payments/pull/8630/commits/1d586031ae3e76888042afabded4b98d0ef39ce2